### ISSUE TITLE
Audio player in user message frame always goes with the default command

### ISSUE THREAD

**Issue by ArturWyszomirski** [ArturWyszomirski/Plugin.Maui.Chat#14]
```csharp
[File: CustomizedChatPage/ViewModel]
```
AudioPlayer is used in three places: user entry, sent and received message. In the sent and received message custom method works as it should (viewmodel's PlayAudioCommand is fired). However AudioPlayer nested in the frame above the user message entry always fires the default method. Tracking shows that for some reason when AudioPlayer is resolved the property bound to StartStopCommand is null and in consequence the default value is set to StartStopCommand:
```csharp
[File: AudioPlayer.xaml.cs]
private void OnStartStopCommandChanged(object newValue)
{
    if (newValue is null) 
        StartStopCommand = new Command(async () => await StartStopAudioAsync(Source));
}
```
Worth noting is that the same property is resolved different (not pointing to null but to viewmodel's command) when AudioPlayer is nested in ContentView.Resources (sent and received messages).

Tried below in Chat.xaml without any change in behavior:
```xaml
StartStopCommand="{Binding AudioPlayerCommand, Source={RelativeSource AncestorType={x:Type ContentPage}}}"
```

Steps to reproduce:
1. Set a breakpoint in AudioPlayer.xaml.cs\StartStopCommand
2. Click audio record on CustomizedChat page
3. Close the alert
4. Click audio player that showed-up above user entry
5. Instead of Ukulele Song the default command is fired.

**Comment by ArturWyszomirski** [comment]
Workaround with [Chat.xaml.cs]:
```csharp
protected override void OnBindingContextChanged()
{
    base.OnBindingContextChanged();

    // workaround for StartStopCommand being null when AudioPlayer resolved (issue #14)
    audioPlayer.StartStopCommand = AudioPlayerCommand;
}
```

**Comment by ArturWyszomirski** [comment]
Audio player service has been rebuild and issue is no longer valid.
